doc/classes/Object.xml

```diff
@@ -693,6 +693,7 @@
 			<return type="int" />
 			<description>
 				Returns the object's unique instance ID. This ID can be saved in [EncodedObjectAsID], and can be used to retrieve this object instance with [method @GlobalScope.instance_from_id].
+				[b]Warning:[/b] This ID is specific to the running session. This means that it won't match the same object if sent over a network, or loaded from a file at a later time.
```

Comment:
We could go with "isn't guaranteed to match" as it's technically possible to match up, but I think it's best to be negative here and just say they won't (they won't in general, a lot of things would have to match for them to match up in any non-toy setup) to avoid anyone just trusting to luck